### ISSUE TITLE
Ensure unit developers and QAs are conscious of time 

### ISSUE THREAD

**Issue by PedroGFonseca** [LDSSA/curriculum-development#27]
# Ensure unit developers and QAs are conscious of time 

## Context

One issue we had last year was that the time a LU took to consume was highly inconsistent. Both the Learning notebooks and Exercise notebooks contribute to this lack of consistency. 

## Detailed Description

For instructors, everything is fast and easy [[1]](https://imgs.xkcd.com/comics/wikipedian_protester.png). That can lead us to drastically underestimate how long it takes for a student to (1) read and understand materials, and in particular (2) execute exercises. 

Instructors have historically tended to exceed the 45 minutes for reading + consuming of SLUs, often by adding content to make it "a bit challenging at least". 

Having good guidelines for how long something should be consumable by newbies, and having the QAs enforce those could help with consistency. 

Messaging such as "don't worry if this seems insultingly easy to you" may also help instructors calibrate the difficulty. 

## Possible Implementation
- Add guidelines to the [contributing guidelines](https://github.com/LDSSA/batch3-instructors/blob/master/CONTRIBUTING.md)
- Ensure QAs are aware and empowered to block an exercise for taking too long to execute by newbies
- Ensure QAs check the reading time of learning notebooks

**Comment by dominguezmaria** [comment]
@PedroGFonseca currently, the contributing guidelines are more meant to guide the instructors on how to submit their work in the instructors repo. So I've shared this [doc](https://docs.google.com/document/d/1w6JYwmbw5fWL7NuMSOuxL2LSE7krxesC5uKE8IvfGsE/edit?usp=sharing) with them in order to guide them on the SLU preparation.
However, we can add this info to the contributing guidelines in the future, sounds like a good improvement!

Anyways, as a solution for the issue you raised:
* I've added a paragraph about how long should the SLU last for in the instructions doc. Check "SLU time length".
* I've also added added a "call for simplicity" in paragraph "Checking your SLU’s curriculum guidelines"

Then, @gaborsomogyi if you agree with this regarding QA, we can close this issue :)
